### PR TITLE
added llm model embedding experiments

### PR DESCRIPTION
**Pull request by arbindpd96**
Added All the experiment files . test_2.py files mainly have all the models and thats where we are working right now. It has sentiment and semantic analysis both

### CODE REVIEW

File: experiments/test_2.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    try:
+        chat_completion = client.chat.completions.create(
+            messages=[
+                {
+                    "role": "user",
+                    "content": prompt,
+                }
+            ],
+            model="gpt-4",
+            temperature=0.3
```
Comment (chentschel) [inline experiments/test_2.py:59]: Why would we use a temp=0.3 here @arbindpd96 ?

File: experiments/test_2.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        )
+
+        return chat_completion.choices[0].message.content
+    except Exception as e:
+        return e
+
+
+def getAttr(plot):
```
Comment (chentschel) [inline experiments/test_2.py:67]: We should move / modularize this common pieces / (prompts/test data/config statics)

File: experiments/test_2.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+data_3 = "Taanshi, a young vibrant head of marketing at a company, decides to run for Prime Minister of the country. Her ambitions are met with problems from all over but she doesn't give up and ends up becoming Prime Minister of the company."
+data_4 = "On his dead bead, Anubhav, a tech giant, decides to name his property to his employee, Mohan, who has planned this all along. Mohan faces legal problems but finally becomes the head of the tech company and owns the property."
+data_5 = "Akshat, after loosing his father, starts seeing him in his dreams telling him to sleep calmly. But his sleep is disturbed to an extent that he ends up believing his father is alive. Later he kills himself and sleeps forever."
+data_6 = "Ranbir sees his childhood friend after a long time Alia and falls in love. Ranbir couldn't find her address and went for a quest of finding her address and confessing his love. Finally he met an accident while searching for Alia and the saviour was Alia and they finally met got married."
+
+
+def compare(plot1, plot2, casemessage):
```
Comment (chentschel) [inline experiments/test_2.py:96]: What would be the output here / or test hypotesis we are trying to prove?

**Review by chentschel — COMMENTED** [review]
@arbindpd96 left some comments as talked

File: experiments/test_2.py (hunk 1 above)
Comment (arbindpd96) [reply experiments/test_2.py:59]: With a temperature setting of 0.3, gpt enhance the creativity of the response which was not required and the response was long as well which was not suitable for our sentiment analysis .

File: experiments/test_2.py (hunk 2 above)
Comment (arbindpd96) [reply experiments/test_2.py:67]: Yes we can do it moving forward.

File: experiments/test_2.py (hunk 3 above)
Comment (arbindpd96) [reply experiments/test_2.py:96]: data1, data2 are same kind of movie. Same for data 3,data4. but data5 and data6 are very different movie. So comparison of data5 and data6 with any other plot should give us low similarity percentage.

**Pull request closed without merging**